README.md

```diff
@@ -225,4 +220,25 @@ Each directory will be created with `cl-keytool.jar`, `cl-wallet.jar`, and a P12
 **In `code/data-l1`:**
 -   data-l1.jar     // The executable for the dL1 layer
 
+
+### `hydra remote_start`
+
+This method initiates the remote startup of your metagraph in one of the available networks: testnet, integrationnet, and mainnet.
+
+To commence the remote startup of the metagraph, we utilize the parameters configured on Euclid. The startup process unfolds as follows:
+
+1.  Termination of any processes currently running on the metagraph ports, which by default are 7000 for ml0, 8000 for cl1, and 9000 for dl1.
+2.  Relocation of any existing logs to a folder named `older-logs`, residing within each layer directory: `metagraph-l0`, `currency-l1`, and `data-l1`.
+3.  Initiation of the `metagraph-l0` layer, with `node-1` designated as the genesis node.
+4.  Initial startup as `genesis`, transitioning to `rollback` for subsequent executions. To force a genesis startup, utilize the `--force_genesis` flag with the `hydra remote_start` command.  This will move the current `data` directory to a folder named `older-datas`.
+5.  Detection of missing files required for layer execution, such as `:your_file.p12` and `metagraph-l0.jar`, triggering an error and halting execution.
+6.  Following the initiation of `metagraph-l0`, the l1 layers, namely `currency-l1` and `data-l1`, are started. These layers only commence if their respective JARs are present in the directories. Unlike `metagraph-l0`, the absence of JARs does not prompt an error but rather prevents startup.
```

Comment:
What would be the case where the jars aren't present?

Reply:
The layers won't start, only the metagraph-l0 will start. The snapshots will be produced but without l1 layers